release/script/release-notes/README.md

```diff
@@ -0,0 +1,19 @@
+# Generate Release Notes
+
+To help generate release notes, you can use the following script.
+It will produce a formatted file, but you need to organize the content and possibly reword or remove.
+
+You need two tools installed:
+
+* [GitHub CLI](https://cli.github.com/)
+* Python 3
+
+Run:
+
+```
+./generate-release-notes.sh v2.7
```

Comment:
Nit: generate-initial-release-notes.sh

Reply:
@chenqi0805 , Thank you for catching this. That was the original name I used and then renamed it to help clarify that they are just a starting point. I updated the PR.